pinocchio-counter/README.md

```
@@ -28,13 +28,13 @@ cargo build-sbf
 ## Instructions
 
 ### 0: InitializeCounter
-Initialize a counter PDA to 0.
+Initialize a counter PDA to 0. Payload: `bump` (u8).
 
 ### 1: IncreaseCounter
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)